index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="#" method="POST">
+
+          <ul class="form-content">
+            <li> <label><input type="text" name="fullname" placeholder="Full name"
+              maxlength="30" required></label></li>
+            <li><label><input type="email" name="email" placeholder="Enter your E-Mail address"></label></li>
```

Comment:
- [ ] Please kindly add the `required` attribute to this `input` in order to get consistent contact information in your form.